mkdocs.yml

```diff
@@ -0,0 +1,13 @@
+site_name: EnergySage Design System docs
+repo_url: https://github.com/EnergySage/es-ds
+edit_uri: edit/main
+
+nav:
+  - Home: README.md
+
+docs_dir: docs
```

Comment:
this dir doesn't seem to exist... what's this going to end up doing, if anything?

Reply:
yes.. I was removing that from the file on the other places, but forgot to remove this one... I'm replacing it with ./ root folder. Also, this mkdocs file is beginning of tech docs, we need to do more here regarding tech docs. To add that docs folder and add documentation files inside.